content/learn/migration-guides/0.16-to-0.17.md

```diff
@@ -1625,13 +1625,13 @@ Instead, all id filtering methods were unified into generic `deny_by_ids/allow_b
 
 ## Other affected APIs
 
-| 0.16 | 0.17 |
-| - | - |
-| `EntityWorldMut::clone_with` | `EntityWorldMut::clone_with_opt_out` `EntityWorldMut::clone_with_opt_in` |
+| 0.16                                   | 0.17                                                                                         |
+| -------------------------------------- | -------------------------------------------------------------------------------------------- |
```

Comment:
not sure this is intentional? doesn't matter though, life's too short to worry about markdown formatting :)

Reply:
This was format-on-save's doing!